web/packages/api/src/environment.ts

```diff
@@ -317,7 +317,7 @@ export const SNOWBRIDGE_ENV: { [id: string]: SnowbridgeEnvironment } = {
                     {
                         id: "WETH",
                         address: "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
-                        minimumTransferAmount: 1n,
+                        minimumTransferAmount: 15_000_000_000_000n,
```

Comment:
Needs to be added on line as well.
https://github.com/Snowfork/snowbridge/pull/1234/files#diff-0c721ef12fe6fb53cd42b43781e67cd2000aa1927affa80e849954d3cdca186cR371

Reply:
It doesnt have to `AH->Ethereum` direction as well. But currently you have added it to `AH->Ethereum` and not `Ethereum->AH`. So it reversed in the PR. So location in the config is configured with the tokens it can receive on its chain i.e `...->AH`. So you would go to AH location, and modify the minimum there if you want to set the minimum `Ethereum->AH`.

I would do it in both directions because I want to unify this config at some point. The issue is that some tokens such as MYTH are only allowed to be sent to Mythos and not back to Ethereum. But that is an edge case. Ideally I want to get this from onchain first if it is set.

Reply:
Thanks for explaining! Let me know if this works: https://github.com/Snowfork/snowbridge/pull/1234/commits/dbb15c5416fa64806e220bbbbf78d7d5d271546e